pkg/tuned/dialog-view.jsx

```diff
@@ -77,7 +77,7 @@ var DialogFooter = React.createClass({
     },
     primary_click: function(e) {
         // only consider clicks with the primary button
-        if (e !== undefined && e.button !== 0)
+        if (!e || e.button !== 0)
```

Comment:
This isn't the same and a similar case is in https://github.com/cockpit-project/cockpit/pull/4225/files#diff-f26c25487c3bd53f0b5f36fa27e365b4R99.
Since it's feasible that this function is called without the event (`e`) parameter, it should still run that way.
How about `if (e && e.button !== 0)`? That would be equivalent to the original condition.